ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/ENRResponsePacketDataTest.java

```diff
@@ -29,6 +32,9 @@
 import org.junit.Test;
 
 public class ENRResponsePacketDataTest {
```

Comment:
Same as before, let's keep the `secp256k1` as key for now.

Reply:
:+1: